### ISSUE TITLE
Superset ALERT & REPORTS not working 

### ISSUE THREAD

**Issue by mujohiddin** [apache/superset#28686]
### Bug description

Hello there. my project is last september version but i have problem with ALERT & REPORTS how to fix it. I don't know how to fix it. help me

### How to reproduce the bug

1

### Screenshots/recordings

![photo_2024-05-24_09-58-21](https://github.com/apache/superset/assets/69301931/6cd10e9b-aa50-456b-925e-bedd88f328df)

Could this be the cause of the problem?

### Superset version

master / latest-dev

### Python version

3.9

### Node version

16

### Browser

Chrome

### Additional context

1

### Checklist

- [ ] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [ ] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [ ] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by fzhan** [comment]
@mujohiddin what does your celery status say?

**Comment by gaful** [comment]
Hello, 
It seems that you have 2 issue : 
1. issue with sending email : what does your system logs says ? they are usually quite usefull. 

2. issue with capturing image : from the documentation https://superset.apache.org/docs/configuration/alerts-reports/

> You must install a headless browser, for taking screenshots of the charts and dashboards. Only Firefox and Chrome are currently supported.
> If you choose Chrome, you must also change the value of WEBDRIVER_TYPE to "chrome" in your superset_config.py.
> 
> Note: All the components required (Firefox headless browser, Redis, Postgres db, celery worker and celery beat) are present in the dev docker image if you are following [Installing Superset Locally](https://superset.apache.org/docs/installation/docker-compose/). All you need to do is add the required config variables described in this guide (See Detailed Config).

By the way I would recommend to make it works for csv or txt first and then moove to image

**Comment by sfirke** [comment]
Yes please see the docs that @gaful linked.  I'm going to convert this to a Discussion, as it is not a bug report of a problem with Superset but instead a request for help getting the configuration set up.  Try searching GitHub Discussions and Issues for that error message, it has come up before, e.g. https://github.com/apache/superset/issues/23057.  It has also been addressed in the Slack chat that is open to the public.
